internal/config/validate.go

```diff
@@ -327,9 +327,15 @@ func (c *Pipeline) Validate() error {
 	var result error
 
 	for _, stepRaw := range c.StepRaw {
-		if stepRaw == nil || stepRaw.Use == nil || stepRaw.Use.Type == "" {
+		if stepRaw == nil && stepRaw.PipelineRaw == nil {
 			result = multierror.Append(result, fmt.Errorf(
-				"step stage with a default 'use' stanza is required"))
+				"step stage with a default 'use' stanza or a 'pipeline' stanza is required"))
+		} else if stepRaw.Use != nil && stepRaw.PipelineRaw != nil {
+			result = multierror.Append(result, fmt.Errorf(
+				"step stage with a 'use' stanza and pipeline stanza is not valid"))
```

Comment:
This message could be clearer to express that you can't "use" a plugin _and_ have a nested "pipeline" stanza. (At least that's how I understood this code?)

Reply:
That's correct!